draft-ietf-quic-recovery.md

```diff
@@ -213,14 +213,17 @@ or negotiated in order to better suit a variety of environments.
   a packet lost.  In fraction of an RTT.
 
 * kMinTLPTimeout: 10ms
- Minimum time in the future a tail loss probe alarm may be set for.
+  Minimum time in the future a tail loss probe alarm may be set for.
```

Comment:
The extra ':' is correct -- it's Markdown formatting for a [definition list](https://kramdown.gettalong.org/syntax.html#definition-lists).